GEOSadas currently un-CMakeable

Per @gmao-jstassi:

```
CMake Error at src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSsuperdyn_GridComp/FVdycoreCubed_GridComp/CMakeLists.txt:25 (get_target_property):
  get_target_property() called with non-existent target "GFDL_fms_r4".


CMake Error at src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSsuperdyn_GridComp/FVdycoreCubed_GridComp/fvdycore/CMakeLists.txt:53 (get_target_property):
  get_target_property() called with non-existent target "GFDL_fms_r4".


CMake Error at src/Components/@GEOSgcm_GridComp/GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSmoist_GridComp/CMakeLists.txt:13 (get_target_property):
  get_target_property() called with non-existent target "GFDL_fms_r4".

```
The issue is that GMAO_Shared 1.0.8 moved FMS. Fix is incoming.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@ cmake_policy (SET CMP0054 NEW)
 
 project (
   GEOSadas
-  VERSION 5.25.0
+  VERSION 5.25
   LANGUAGES Fortran CXX C)  # Note - CXX is required for ESMF
 
 if ("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")

--- a/Externals.cfg
+++ b/Externals.cfg
@@ -34,6 +34,13 @@ local_path = ./src/Shared/@MAPL
 tag = v1.1.6
 protocol = git
 
+[FMS]
+required = True
+repo_url = git@github.com:GEOS-ESM/FMS.git
+local_path = ./src/Shared/@GFDL_fms
+tag = geos/orphan/v1.0.0
+protocol = git
+
 [GEOSana_GridComp]
 required = True
 repo_url = git@github.com:GEOS-ESM/GEOSana_GridComp.git

--- a/src/Shared/.gitignore
+++ b/src/Shared/.gitignore
@@ -1,3 +1,4 @@
 /@GMAO_Shared
 /@MAPL
 /@NCEP_Shared
+/@GFDL_fms

--- a/src/Shared/CMakeLists.txt
+++ b/src/Shared/CMakeLists.txt
@@ -3,3 +3,8 @@ esma_add_subdirectories (
   @GMAO_Shared
   @NCEP_Shared
   )
+
+# Special case - GFDL_fms is built twice with two
+# different precisions.
+add_subdirectory (@GFDL_fms GFDL_fms_r4)
+add_subdirectory (@GFDL_fms GFDL_fms_r8)

